hr_holidays_public/models/hr_holidays_public.py

```diff
@@ -55,42 +56,37 @@ def name_get(self):
 
     @api.model
     @api.returns("hr.holidays.public.line")
-    def get_holidays_list(self, year, employee_id=None):
+    def get_holidays_list(self, date_from, date_to, employee_id=None):
```

Comment:
Please don't change API. You can make `year` a keyword argument, and add `date_from` and `date_to` as other keyword arguments, so they are optional, and use the most convenient one.